lib/elixir_jobs/email.ex

```diff
@@ -0,0 +1,31 @@
+defmodule ElixirJobsWeb.Email do
+  #import Bamboo.Email
+  use Bamboo.Phoenix, view: ElixirJobsWeb.EmailView
+
+  def notification_offer_created_html(offer) do
+    emails = ElixirJobs.Users.Admin |> ElixirJobs.Users.Queries.Admin.admin_emails |> ElixirJobs.Repo.all
```

Comment:
In the project we are using Phoenix 1.3.0 with Contexts, so the best way to do this is moving it to a function inside the `Users` context, which all the logic lives.

If you are okay with just getting the administrators and then get their e-mail addresses (so avoid extra code for each case):

```elixir
emails =
  ElixirJobs.Users.list_admins()
  |> Enum.map(&({&1.name, &1.email}))
```